.github/workflows/core-build.yml

```diff
@@ -53,5 +53,5 @@ jobs:
       - name: Build Maplibre GL Core
         run: |
           cmake --version
-          cmake -B build -DCMAKE_BUILD_TYPE=Debug -DMBGL_WITH_CORE_ONLY=ON -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DMBGL_WITH_COVERAGE=ON
+          cmake -B build -DCMAKE_BUILD_TYPE=Debug -DMBGL_WITH_CLANG_TIDY=ON -DMBGL_WITH_CORE_ONLY=ON -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DMBGL_WITH_COVERAGE=ON
```

Comment:
I decided that it is the good place to run clang-tidy checks on CI.

Reply:
Good!